migration/v0.17.x-v0.18.x/remove-velero-backupstoragelocation.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+here="$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"
+
+set -euo pipefail
+
+: "${CK8S_CONFIG_PATH:?Missing CK8S_CONFIG_PATH}"
+
+# Delete unused aws backupstoragelocation.
+"${here}/../../bin/ck8s" ops kubectl sc delete backupstoragelocation -n velero aws
```

Comment:
This wouldn't work if it is set to gcs.

Maybe just make a simple if statement that checks if `objectStorage.type` set to `aws` else if it's set to `gcs` delete `gcs`